mantle/Makefile

```diff
@@ -11,6 +11,7 @@ endif
 
 .PHONY: build test vendor
 build:
+	./generate
```

Comment:
But this requires `schematyper` in the path right?  That's going to break everyone who doesn't have it, including `Dockerfile` today.

Reply:
Maybe simplest to vendor it?

We had a similar situation with rpm-ostree and `bindgen` too.

Reply:
Ah, that is true....good catch :)